docs/proposals/ci-in-github-action.md

```diff
@@ -0,0 +1,56 @@
+# Proposal: CI in Github Action
+
+Last updated: 01.11.2021
+
+## Abstract
+
+Description of triggers and user actions to run CSI Baremetal CI.
+
+## Background
+
+We need to create Github Action to perform CSI Baremetal CI tests, 
+because running tests locally requires much hardware resources and developers' time.
+It is also making simpler to validate Pull Requests from fork-repositories.
+
+## Proposal
+
+1. Developer: Add a comment to PR with the following content
+```
+/ci
+operator_branch=branch1
+```
+
+2. Action: Add a comment to PR
+```
+Start CI. Parameters:
+operator_branch=branch1
+```
+
+3. Action: Perform CI suite
```

Comment:
Where do you plan to run CI?

Reply:
It will be a part of action. I want to show user interactions in this proposal

Reply:
I'm asking about target host where you plan to execute CI. How many actions can be executed in parallel? What are the resources available - CPU, Memory, Storage spaces, etc. Do we need to use kind or we can get k8s cluster?

Reply:
We plan to run CI on Github-hosted runners. They have 7 GB RAM and 15 GB disk space ([reference](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners)). I have found no information about limitations of parallel actions, only in minutes. We need to use kind as we have one host only

Reply:
Updated info about environment in the proposal

Reply:
Does CI requires `--privileged`  ?